docs/product-installation-guide/src/main/asciidoc/clustering-dm-install-proc.adoc

```diff
@@ -19,8 +22,8 @@ After you have installed Apache ActiveMQ Artemis and Elasticsearch, complete the
       <niogit_dir_on_shared_nfs>"/>
   <property name="appformer-cluster" value="true"/>
   <property name="appformer-jms-url" value="tcp://<MAIN_NODE_IP>:61616"/>
-  <property name="appformer-jms-username" value="admin"/>
-  <property name="appformer-jms-password" value="admin"/>
+  <property name="appformer-jms-username" value="JMS_BROKER_USER"/>
```

Comment:
I think that there should be brackets to be aligned with other replacements - <JMS_BROKER_USER>